drilldown/templates/drilldown/drilldown.html

```diff
@@ -137,6 +154,28 @@ <h3>{% for fname, flabel in FILTER_NAMES.items %}
         {% endfor %}
     </div>
 
+    [<a onclick="javascript:toggle(document.querySelector('#sample'))">toggle samples</a>]
+    <div id="sample" class="hidden shown-1000">
+        <table>
+            <thead>
+                <td>Name
+                <td>Address
+                <td>City
+                <td>Zipcode
+            </thead>
+            <tbody>
+                {% for v in sample_results %}
+                <tr>
+                    <td>{{ v.data.first_name }} {{ v.data.last_name }}
```

Comment:
DIS-271 asked for

first, middle, last name
race
gender
party
street address and city
phone number